### ISSUE TITLE
Issue with datetime

### ISSUE THREAD

**Issue by CGraabaek** [kevhauma/MessengertoStats#2]
When running the script with a message HTML file, I get this error ` datetime =preDT[1].split('at')`

TypeError: Cannot read property 'split' of undefined
                 at /Users/xxxxx/Desktop/xxxxx/MessengertoStats-master/script.js:103:43

Any ideas?

**Comment by kevhauma** [comment]
can you tell me what format the time is in the HTML file?

**Comment by CGraabaek** [comment]
Here is a snippet of such an HTML file. 

`</style>
<title>Samtale med Oliver Halkjær</title>
</head>

<body>
    <a href="html/messages.htm">Tilbage</a>
    <br />
    <br />
    <div class="thread">
        <h3>Samtale med Oliver Halkjær</h3>Participants:
        <div class="message">
            <div class="message_header">
                <span class="user">Oliver Halkjær</span>
                <span class="meta">23. august 2014 kl. 20:58 UTC+02</span>
            </div>
        </div>
        `

**Comment by CGraabaek** [comment]
<img width="574" alt="screen shot 2018-04-17 at 14 38 21" src="https://user-images.githubusercontent.com/6660378/38870087-04ac571a-424d-11e8-9ed7-5b63e8a61c41.png">

**Comment by kevhauma** [comment]
I will look into a solution for different formats

**Comment by CGraabaek** [comment]
Thank you, that would be awesome!

**Comment by kevhauma** [comment]
it should now work with your time format, 

going to leave this issue open in case it fails with other formats.

**Comment by CGraabaek** [comment]
It works now, thanks man, this is awesome!

**Comment by CGraabaek** [comment]
Just a side node it almost works. 

To get it to work properly i have changed the following lines: 

Line 5 is changed from:
`let monthsArray = ["jan", "feb", "mar", "apr", "may", "jun", "jul", "aug", "sept", "oct", "nov", "dec"]`
To
`let monthsArray = ["jan", "feb", "mar", "apr", "maj", "jun", "jul", "aug", "sept", "okt", "nov", "dec"]`

Line 440 is changed from: 
` case 'may':`
To
` case 'maj':`

Line 450 is changed from: 
` case 'oct':`
To
` case 'okt':`

These changes are to accomodate the different spellings in month across languages.

**Comment by kevhauma** [comment]
There is not really an easy way to implement different languages, 

Might add a config file or something to make it easier.

**Comment by CGraabaek** [comment]
Yes I know, it was just an observation if you wanted to know or other people encounter the same problems.

**Comment by kevhauma** [comment]
i've added a file where people can change the months themselves, 
that should do that trick
